core/tests/test_utils.py

```diff
@@ -241,14 +242,25 @@ def post_json(self, url, payload, csrf_token=None, expect_errors=False,
         if csrf_token:
             data['csrf_token'] = csrf_token
 
-        json_response = self.testapp.post(
-            str(url), data, expect_errors=expect_errors,
-            upload_files=upload_files)
+        json_response = self.post(
+            url, data, expect_errors, expected_status_int, upload_files)
 
-        self.assertEqual(json_response.status_int, expected_status_int)
         return self._parse_json_response(
             json_response, expect_errors=expect_errors)
 
+    def post(self, url, data, expect_errors=False, expected_status_int=200,
```

Comment:
Mm, not sure about this. I think it's better to expose post_json and post_email. Maybe they share a post() or send_post_request() behind the scenes, but that latter method should be private.